docs/docs/30-administration/11-forges/20-github.md

```diff
@@ -1,72 +1,73 @@
 # GitHub
 
-Woodpecker comes with built-in support for GitHub and GitHub Enterprise. To enable GitHub you should configure the Woodpecker server using the following environment variables:
-
-```diff
-# docker-compose.yml
-version: '3'
-
-services:
-  woodpecker-server:
-    [...]
-    environment:
-      - [...]
-+     - WOODPECKER_GITHUB=true
-+     - WOODPECKER_GITHUB_CLIENT=${WOODPECKER_GITHUB_CLIENT}
-+     - WOODPECKER_GITHUB_SECRET=${WOODPECKER_GITHUB_SECRET}
-
-  woodpecker-agent:
-    [...]
+Woodpecker comes with built-in support for GitHub and GitHub Enterprise.
+To use Woodpecker with GitHub the following environment variables should be set for the server component:
+
+```sh
+WOODPECKER_GITHUB=true
+WOODPECKER_GITHUB_CLIENT=YOUR_GITHUB_CLIENT_ID
+WOODPECKER_GITHUB_SECRET=YOUR_GITHUB_CLIENT_SECRET
 ```
 
-## Registration
+You will get these values from GitHub when you register your application.
+To do so, go to Settings -> Developer Settings -> GitHub Apps -> New GitHub App.
 
-Register your application with GitHub to create your client id and secret. It is very important that the authorization callback URL matches your http(s) scheme and hostname exactly with `<scheme>://<host>/authorize` as the path.
+## App Settings
 
-Please use this screenshot for reference:
+- Name: An arbitrary name for your App
+- Homepage URL: The URL of your Woodpecker instance
+- Callback URL: `https://<your-woodpecker-instance>/authorize`
+- Leave "Request user authorization (OAuth) during installation" and "Enable Device Flow" unchecked
+- Leave "Webhook" and "Post Installation" fields empty
```

Comment:
The Webhook url field is required, cannot be left empty.

Reply:
It is set by Woodpecker (as an Oauth app) here:
https://github.com/woodpecker-ci/woodpecker/blob/main/server/api/repo.go#L116

The access token is generated there (in the PostRepo code), so you won't be able to fill in the field when creating Woodpecker as a Github App.